packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -1315,7 +1347,8 @@ namespace Tpetra {
     ///   only, and should never be called by user code.
     void
     setAllIndices (const typename local_graph_type::row_map_type& rowPointers,
-                   const typename local_graph_type::entries_type::non_const_type& columnIndices);
+                   const typename local_graph_type::entries_type::non_const_type& columnIndices,
+                   const bool overwrite=false);
```

Comment:
If we let users set `overwrite` and thus bypass the check, then it seems like we could just get rid of the check and keep the interface the same.  It's not obvious from the interface that the check exists.  Most "users" are actually MueLu developers, so they might not mind just getting rid of the check.

Reply:
Perhaps?  Or I should just take the overwrite parameter out.  It was from an earlier incarnation where I used setAllIndices, I changed in favor of writing the more general resizeForIncomingData.  I’ll have to double check, but I don’t believe it is used anymore

Reply:
That would be even better; thanks!